src/mixins/extensions/Watchers.js

```diff
@@ -13,7 +13,7 @@ export default {
       if (definition.watchers) {
         screen.mixins.push(watchersMixin);
         definition.watchers.forEach((watcher) => {
-          this.addWatch(screen, watcher.watching, `this.queueWatcher(${JSON.stringify(watcher)});`);
+          this.addMounted(screen, 'var that = this; this.$watch("' + watcher.watching + '", function() { that.queueWatcher(' + JSON.stringify(watcher) + '); });');
```

Comment:
Could you try Arrow Function declaration instead the use of `var that = this;`:
```
    this.addMounted(
        screen,
        'this.$watch("' + watcher.watching + '", () => { this.queueWatcher(' + JSON.stringify(watcher) + '); });'
    );
```

Reply:
@caleeli The problem is it's a string so Babel doesn't convert it to ES5